quine-rocksdb-persistor/src/test/scala/com/thatdot/quine/persistor/RocksDbPersistorSpec.scala

```diff
@@ -13,7 +16,7 @@ class RocksDbPersistorSpec extends PersistenceAgentSpec {
   lazy val persistor: PersistenceAgent =
     if (RocksDbPersistor.loadRocksDbLibrary()) {
       val f = Files.createTempDirectory("rocks.db")
-      f.toFile.deleteOnExit()
```

Comment:
I believe this won't work for non-empty directories.

Reply:
Great catch, you are absolutely right -- thanks for the patch!

Reply:
There are two: For this particular case, scalafix is the thing blocking CI -- the `fixall` target runs scalafix and scalafmt, linting and fixing as many issues as possible. If you wish to run exactly what CI does, the task definitions are here: https://github.com/thatdot/quine/blob/main/.github/workflows/ci.yml#L1

Usually, just running the 2.12 command is enough to tell whether the change will pass CI:

`sbt -v 'scala212 test' quine/assembly quine-docs/makeSite 'scalafixAll --check'`